src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,64 @@
+function NewEvaluationForm(options){
+    return `
+    ${CandidateDetailsform()}
+    ${TechnicalLevelPicker()}
+    
+    `
+}
+
+
+function CandidateDetailsform(options={}){
+    return `
+        <form class="main-form" method="POST" action="newEval.php">
+        <div class="data-form">
+            <input type="text" placeholder="Candidate" name="Candidate">
+            <input type="text" placeholder="Interviewer" name="Interviewer">
+            <input type="date">
+        </div>
+     `
+}
+
+function TechnicalLevelPicker(options={}){
+    return`
+    <h1>Technical Level</h1>
+    <div>
+        <table class="table">
+            <tr class="row1">
```

Comment:
I would like to see this data dynamically generated

Reply:
I deleted my file, this is an older version of the file.